docs/UserGuide.md

```diff
@@ -280,6 +280,56 @@ Examples:
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
 [Back To Contents](#table-of-contents)
+
+<hr style="border:2px solid gray">
+
+### Editing the details of the user : `edituser`
+
+Want to change one of your own details, perhaps something was out of date?
+
+Use this command to edit your details easily!
+
+**Syntax:**
+
+`edituser PREFIX/PARAMETER...`
+
+_Here are some important requirements for you to take note:_
+
+* At least one field must be provided.
+
+For the following fields, they are considered a `SuperField`.
```

Comment:
We are planning to change `SuperField` into `FieldGroup`. So change it for this PR as well.